{{ cookiecutter.project_slug }}/config/webpack.base.js

```diff
@@ -87,7 +88,12 @@ module.exports = {
         new WebpackNotifierPlugin({
             title: '{{ cookiecutter.project_name }}',
             skipFirstNotification: true
-        })
+        }),
+        new Dotenv({
```

Comment:
@Struchu I'm unsure if this is a valid approach, could you chime in?

If I understand how this works, `dotenv-webpack` does a text-replace on used environment variables, and includes plain strings in the bundle - which is good. However, that couples the build process with `.env` being present and properly configured.

I'm not particularly fond of using `.env` during webpack builds. I would assume this should rather be a configuration element, just like API url or other configuration aspects of the JS app.

Reply:
@maticomp I think I'm on your side on this. Since the Sentry on JS side is optional I would put the DSN in the `sentry.js` file directly and drop the dependency on the dotenv for webpack.